fluent/syntax/ast.py

```diff
@@ -67,174 +82,175 @@ def visit(value):
 
     def to_json(self):
         obj = {
-            name: to_json(value)
+            u(name): to_json(value)
             for name, value in vars(self).items()
         }
         obj.update(
-            {'type': self.__class__.__name__}
+            {'type': u(self.__class__.__name__)}
         )
         return obj
 
     def __str__(self):
         return json.dumps(self.to_json())
 
 
+class Node(BaseNode):
```

Comment:
can you write a very short docstring that BaseNode=(Node, MetaNode) and what are Nodes and MetaNodes?